data/battle_scripts_1.s

```diff
@@ -370,6 +370,12 @@ gBattleScriptsForMoveEffects::
 	.4byte BattleScript_EffectHit                     @ EFFECT_FICKLE_BEAM
 	.4byte BattleScript_EffectHit                     @ EFFECT_BLIZZARD
 	.4byte BattleScript_EffectHit                     @ EFFECT_RAIN_ALWAYS_HIT
+	.4byte BattleScript_EffectHit                     @ EFFECT_PSYCHIC_NOISE
+
+BattleScript_EffectPsychicNoise::
```

Comment:
Why is this still defined (also in battle_scripts.h) if it's unused?

Reply:
good question.

Reply:
removed